radio/src/hal/module_port.h

```diff
@@ -145,10 +145,16 @@ inline uint8_t modulePortGetType(const etx_module_driver_t& d) {
 }
 
 inline const etx_serial_driver_t* modulePortGetSerialDrv(const etx_module_driver_t& d) {
+#if defined(SIMU)
+  if (d.port == nullptr) return nullptr;
+#endif
```

Comment:
It's probably worth adding this code even for non-simu, IMHO.

Reply:
This is changed by @philmoz instead, any comments.